snippets/csharp/VS_Snippets_CLR/Process_GetProcessesByName2_2/CS/process_getprocessesbyname2_2.cs

```diff
@@ -38,7 +38,7 @@ public static void Main(string[] args)
             foreach (Process myProcess in myProcesses)
             {
                 Console.Write("Process Name : " + myProcess.ProcessName + "  Process ID : "
-                + myProcess.Id + "  MachineName : " + myProcess.MachineName + "\n");
+                    + myProcess.Id + "  MachineName : " + myProcess.MachineName + "\n");
```

Comment:
This would be a good candidate for using string interpolation. I'll approve the PR, but wait to merge if you want to make that change.